### DISCUSSION TITLE
Should we add a feature that allows a block in the side bar to render above the content for small screens?

### DISCUSSION THREAD

**Discussion by AmnestyAM** [amnestywebsite/humanity-theme#483] [Feature Requests]
For petitions, it is often requested that the users want the form to appear above the copy when on mobile, so the action is not at the very bottom of the page.

My guess is we could do this on the petition single as we have a template for this, but it might be harder on a standard post or page. 

What do you think?

**Comment by carolinejcourtney** [comment]
I think this is a great idea and would probably be most valuable on petitions anyways. Would we consider a design that puts the sidebar content in a block that sticks to the bottom of the screen on mobile and can be open/closed? That way people can easily navigate to the form from anywhere on the page on mobile rather than having to scroll back up or down.



Caroline Courtney

Global Website Editor

Website Operations Team Lead

They/Them

________________________________
From: Amnesty AM ***@***.***>
Sent: Thursday, November 14, 2024 11:04 AM
To: amnestywebsite/humanity-theme ***@***.***>
Cc: Subscribed ***@***.***>
Subject: [amnestywebsite/humanity-theme] Should we add a feature that allows a block in the side bar to render above the content for small screens? (Discussion #483)


⚠ CAUTION External Sender Exercise caution opening links or attachments. Do not provide login details.


For petitions, it is often requested that the users want the form to appear above the copy when on mobile, so the action is not at the very bottom of the page.

My guess is we could do this on the petition single as we have a template for this, but it might be harder on a standard post or page.

What do you think?

—
Reply to this email directly, view it on GitHub<https://github.com/amnestywebsite/humanity-theme/discussions/483>, or unsubscribe<https://github.com/notifications/unsubscribe-auth/AGHUUYWGHCNBEU3CDPU7QAD2AR7TFAVCNFSM6AAAAABRYT5XC2VHI2DSMVQWIX3LMV43ERDJONRXK43TNFXW4OZXGQ4DCOBWGE>.
You are receiving this because you are subscribed to this thread.Message ID: ***@***.***>

DISCLAIMER
This email has been sent by Amnesty International Limited (a company registered in England and Wales limited by guarantee, number 01606776 with a registered office at 1 Easton St, London WC1X 0DW). Internet communications are not secure and therefore Amnesty International does not accept legal responsibility for the contents of this message. If you are not the intended recipient you must not disclose or rely on the information in this e-mail. Any views or opinions presented are solely those of the author and do not necessarily represent those of Amnesty International unless specifically stated. Electronic communications including email might be monitored by Amnesty International for operational or business reasons..

**Comment by SophieAmnesty** [comment]
This has come up in the recent user survey: 

> I would suggest to make it easy when we want make some changes (especially on the position of the petition form, put it in the top of the article on the mobile to make the user experience better)

**Comment by SophieAmnesty** [comment]
Just to add to this discussion, the Amnesty Indonesia site have recently added the petitions plugin and moved the iframe to the left hand side so that it would appear first on mobile. 

![image](https://github.com/user-attachments/assets/4891ecae-3bec-4add-912c-3902a860922c)

Only problem with this is that there's no padding between the iframe and the text now so doesn't look great on desktop. 

If we added some padding to the iframe perhaps this could be a quick fix/workaround for the issue above? @AmnestyAM what do you think?

**Reply by SophieAmnesty** [reply]
p.s. I have managed to add a spacer: 

![image](https://github.com/user-attachments/assets/640341ad-fc36-4fd6-9953-ffb9e799ce10)
 
![image](https://github.com/user-attachments/assets/4e319f8c-0c78-462d-bd65-577891df2897)
